.gitignore

```diff
@@ -0,0 +1,8 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+**__pycache__/
+**db.sqlite3
```

Comment:
add blank line at the end of each file
also you pushed .idea folder

Reply:
I didn't understand. I shouldn't add (.idea) to .gitignore file?